### DISCUSSION TITLE
Quesetion about LiteLLM integration with AWS Sagemaker endpoint

### DISCUSSION THREAD

**Discussion by luciameng1989** [BerriAI/litellm#2648] [Q&A] [answered]
Hi,

If I use LiteLLM to call a AWS Sagemaker endpoint, does my data just transmit between my local machine and the AWS endpoint? Is there any risk that the data could be sent to any third party?

**Comment by ishaan-jaff** [comment, marked as answer]
Hi @luciameng1989 
> does my data just transmit between my local machine and the AWS endpoint

Yes pretty much 

> Is there any risk that the data could be sent to any third party?

No risk. We used to have some telemetry but the latest versions of litellm have telemetry removed 

@luciameng1989 I'd love to hop on a call and learn how we can improve litellm for you. Link to my cal for your convenience : https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat?month=2024-03
